wuerthcs/flywheel-frontend
# DRAFT: integration of react UI Library

this PR is in draft mode and can't be merged yet because it is blocked by this ticket https://app.shortcut.com/wuerthcs/story/31254/plan-web-update-node-version-to-node-js-18-17-1-for-docker-file
you may see some issues related to UI Library while running "npm i" those issues will be resolved by the above story 31254

Testing steps:
Run the app
Make sure that React UI Library is  integrated and works as expected(a component from the UI Library was used in the displayed text for testing purposes)
